azure-advanced/Packer/ansible-apim/dev/group_vars/apim-analytics.yml

```diff
@@ -0,0 +1,200 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2020 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#----------------------------------------------------------------------------
+
+wso2_group: wso2
+wso2_user: wso2carbon
+product_name: wso2am-analytics
+product_version: 3.1.0
+target: /mnt
+product_package_location: files
+misc_file_location: "{{ product_package_location }}/misc"
+misc_file_copy_location: "{{ target }}/misc"
+security_file_location: "{{ product_package_location }}/security"
+jdbc_driver: mysql-connector-java-5.1.48-bin.jar
```

Comment:
@rasanjaya85 shall any particular reason why we do not use the version `8` range of JDBC connectors?

Reply:
No. This version is newer than the once comes with the Ansible playbook.